pkg/cmd/deprecation/deprecated.go

```diff
@@ -171,45 +54,15 @@ var deprecatedCommands map[string]deprecationInfo = map[string]deprecationInfo{
 		date: "Feb 1 2020",
 		info: "This is now replaced by apps",
 	},
-	"step create jenkins": {
-		date: "Feb 1 2020",
-	},
-	"step create install values": {
-		replacement: "jx step verify ingress",
-		date:        "June 1 2020",
-		info:        "the command stays, its just been renamed to be with the other 'jx step verify ...' commands to improve the UX",
-	},
 	"step post": {
 		date: "Feb 1 2020",
 	},
-	"step pre": {
-		date: "Feb 1 2020",
-	},
-	"step credential": {
-		date: "Jun 1 2020",
-		info: "This is replaced by Tekton's mounted secrets",
-	},
-	"step git credential": {
-		date: "Jun 1 2020",
-		info: "This is replaced by Tekton's mounted secrets",
-	},
-	"step split monorepo": {
-		date: "Feb 1 2020",
-	},
 	"step nexus drop": {
 		date: "Feb 1 2020",
 	},
 	"step nexus release": {
 		date: "Feb 1 2020",
 	},
-	"console": {
-		replacement: "jx ui",
-		date:        "Jun 1 2020",
-		info:        "Classic Jenkins console will be replaced by Jenkins X UI app",
-	},
-	"controller workflow": {
-		date: "Feb 1 2020",
-	},
 }
```

Comment:
I don't think the changes in this commit are relevant to #7090, would it be better to move it into a separate PR?

Reply:
Can do. It was just a cleanup/chore prior to me adding the deprecation of the `jx [create|delete] vault`. I kept it separate since as you say it could potentially be a separate PR. Same for the first commit, which is also more of a cleanup.

Reply:
I think we should move both this commit and that one to separate PR's, it's more work and frustrating. But it will set a good example for others on how to create PR's that are focussed on solving one problem really well.

Reply:
I feel like I'm nit picking your work and it could certainly be argued that the value provided is not great. But it will set a great precedence.